controllers/convoController.js

```diff
@@ -104,10 +108,36 @@ const fetchAllMyConversationsGet = async (req, res) => {
 
 // Get a specific conversation and its messages
 const fetchConversationGet = async (req, res) => {
+	const { id, lang } = req.query;
+
 	try {
 		// Finding all messages associated with a specific conversation
-		const conversation = await Message.find({ conversationID: req.params.id });
-		res.send(conversation);
+		let conversation = await Message.find({ conversationID: id });
```

Comment:
ah we may want to rethink our Message model because instead of translating the same content each time you update or grab your conversation, we want a more efficient way of doing this - or else you will reach your rate limit for your google api very soon!

Reply:
we can talk about this tomorrow in our call!

Reply:
> ah we may want to rethink our Message model because instead of translating the same content each time you update or grab your conversation, we want a more efficient way of doing this - or else you will reach your rate limit for your google api very soon!

Oh yeah huh, that's a good point. Looking forward to talking more about it tomorrow!

Reply:
at most translate each message once per message per language